### DISCUSSION TITLE
cursor_inactive_timeout in hyprland-git

### DISCUSSION THREAD

**Discussion by se7en-x230** [prasanthrangan/hyprdots#1475] [General]
cursor {
inactive_timeout = 3 # Hyprland uses second
}

is the new format 
not
general {
cursor_inactive_timeout = 3
}

in case someone cares

**Comment by kRHYME7** [comment]
Noted! Thanks!  is the hyprland-git okay now. I always experience crashes yesterday

**Comment by se7en-x230** [comment]
I had no problems
